src/ecalc/libraries/libecalc/common/libecalc/common/dump.py

```diff
@@ -0,0 +1,56 @@
+from functools import wraps
+from pathlib import Path
+from typing import Optional, Any, Callable
+
+from libecalc.common.logger import logger
+
+unique_id: int = 1
+"""
+global static id to increase by one for each file being dumped
+"""
+
+
+def dump(file_dump: Path, captured: Any):
```

Comment:
```suggestion
def dump(output_folder: Path, content: Any):
```
Naming here was a bit confusing for me

1. Was not clear to me that file_dump was a folder

2. or captured content?